mapit_mysociety_org/templates/account/delete.html

```diff
@@ -14,7 +14,7 @@
         <form method="post" action="{% url "account_delete" %}" autocapitalize="off">
             {% csrf_token %}
             <div class="account-form__field account-form__field--submit">
-                <button type="submit" class="btn">{% trans "Delete My Account" %}</button>
+                <button type="submit" class="btn btn--danger">{% trans "Delete My Account" %}</button>
```

Comment:
Oh, I didn't see the delete account page! I think, for simplicity sake, we should change that to do the same as cancel subscription page ie. keep the subscription/account to the end of the current month and then delete it all.